dlrs/main/classes/RollupCalculateJob.cls

```diff
@@ -113,19 +113,31 @@ public with sharing class RollupCalculateJob implements Database.Batchable<sObje
       // Retrieve organization info to put in the email
       Organization org = [SELECT Id, Name FROM Organization];
 
+      //Getting rollup name
+      String rollupName = rollup.isEmpty() ? '' : rollup[0].Name;
+
       // Errors are emailed to the user that scheduled this job
       Messaging.SingleEmailMessage mail = new Messaging.SingleEmailMessage();
       mail.setToAddresses(new List<String>{ UserInfo.getUserId() });
```

Comment:
I know you didn't write this but can you try and make a change here (in both files)

I think the org limits using `setToAddresses` is lower than using `setTargetObjectId`, I think I'd like to see it switched while we're working on this file.

Something like:
```apex
      mail.setTargetObjectId(UserInfo.getUserId());
      mail.setSaveAsActivity(false); // required when sending to user
```